### DISCUSSION TITLE
Report vulgar usernames

### DISCUSSION THREAD

**Discussion by myfirstnameispaul** [community/community#18036] [Other Feature Feedback, Questions, & Ideas]
I admit I can't find what guidelines there are for vulgar usernames, but I would like to request a report option for vulgar usernames.

It is not always the case that a user with a vulgar username is exhibiting abusive behavior as to justify blocking them, and may actually be providing useful behavior.

Nonetheless, not everyone wants to see vulgar usernames in their inbox, so it would be nice if we could report vulgar usernames and then give the user some reasonable time period to revise their usernames.

**Comment by ettaboyle** [comment]
Hi @myfirstnameispaul 👋🏻     

We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

* [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
* [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
* [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
* [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you,

The GitHub Community Forum Team

**Reply by myfirstnameispaul** [reply]
If I report a user, the interface only permits reporting for content, but the user does not have content to report.

It is **ONLY** the username, and there is no way to report only the username, itself.
